mevolib/inference/__init__.py

```diff
@@ -137,6 +138,7 @@ def get_phylogeny(
         )
     # Get the command line to run in order to get the resultant phylogeny
     infile_path = Path(infile).absolute()
+    print("Hzlo: ",Path(infile_path))
```

Comment:
Is this line necesary? A few lines later, you already print the complete command.

Reply:
That's an old commit